src/context/AgencyContext.js

```diff
@@ -1,16 +1,67 @@
 import { createContext, useState } from "react";
+import axios from "axios";
+import { APIUrl } from "../config/config";
 
 export const AgencyContext = createContext({});
 
 export default function AgencyContextProvider({ children }) {
-  const [selectedAgency, setSelectedAgency] = useState();
+  const [selectedAgency, setSelectedAgency] = useState({ company_id: 1 });
   const [pop, setPop] = useState(false);
+  const [cityList, setCityList] = useState({});
+  const [error, setError] = useState(null);
+  const [state, setState] = useState({});
+  const header = {
+    headers: {
+      authorization: `Bearer ${localStorage.token}`,
+      "Content-Type": "application/json",
+    },
+  };
+
+  async function submitData() {
```

Comment:
Maybe make this: updateAgency and another function postAgency

Reply:
👍